transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/utility/PathUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.qpay.transactionhistorymanager.utility;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
```

Comment:
let's don't use it https://www.notion.so/Remove-UtilityClass-annotation-897291a13b9449f7af2221392e46e736